packages/server/lambdas/alfresco/searchQueryBuilder/luceneQueryBuilder.ts

```diff
@@ -0,0 +1,78 @@
+import { IMimeSearchParameter, IModifiedSearchParameter, Paging, SearchParameter } from './types';
+
+const mimeTypesMapping = {
+  MSWORD: ['"application/msword"', '"application/vnd.openxmlformats-officedocument.wordprocessingml.document"'],
+  PDF: ['"application/pdf"'],
+};
+
+const DIVIDER = ':';
+const SEARCH_START = '+@cm\\' + DIVIDER;
```

Comment:
Or could combine into one single:  `+@cm\\:${filterType}:${filterContent}` for example?